ang/test/mocks/data/cases.data.js

```diff
@@ -0,0 +1,75 @@
+(function () {
+  CRM.civicase = {};
```

Comment:
Shouldn't this be `CRM.test.civicase` to avoid variable name conflicts?

Reply:
No, because the code base expects `CRM.civicase.caseActions` to be present in the runtime. In browsers it is populated using backend, thats we need to mock it for unit tests.

Reply:
And these variables are essential for the code to work, for example https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase/Actions.js#L295.
If we dont populate the values, error will be thrown.